modules/slugs/includes/qtranslate-slug-admin.php

```diff
@@ -430,6 +430,40 @@ function qts_show_edit_term_fields( $term ) {
     <?php
 }
 
+/**
+ * Display link to slugs settings for add custom tax admin page (e.g. WooCommerce product attributes).
+ *
+ */
+function qts_show_add_tax_slugs_option_link() {
+    ?>
+    <div class="form-field term-slug-wrap">
+        <label><?php _e( 'Slugs per language', 'qtranslate' ) ?></label>
+        <?php
+        //TODO: link destination should not be hardcoded here, but currently $options_uri property is private in QTX_Admin_Settings class (base options page) and module id is hardcoded independently from module definitions in QTX_Admin_Module class (module href).
+        echo sprintf( "<p>" . __( 'Multilanguage slugs can be set up in <a href="%s">slugs module settings</a> once the new item is added.', 'qtranslate' ) . "</p>", admin_url( 'options-general.php?page=qtranslate-xt#slugs' ) );
+        ?>
+    </div>
+    <?php
+}
+
+/**
+ * Display link to slugs settings for edit custom tax admin page (e.g. WooCommerce product attributes).
+ *
+ */
+function qts_show_edit_tax_slugs_option_link() {
+    ?>
+    <tr class="form-field term-slug-wrap">
+        <th><?php _e( 'Slugs per language', 'qtranslate' ) ?></th>
+        <td>
+            <?php
+        //TODO: link destination should not be hardcoded here, but currently $options_uri property is private in QTX_Admin_Settings class (base options page) and module id is hardcoded independently from module definitions in QTX_Admin_Module class (module href).
+        echo sprintf( "<p>" . __( 'Multilanguage slugs can be set up in <a href="%s">slugs module settings</a>', 'qtranslate' ) . "</p>", admin_url( 'options-general.php?page=qtranslate-xt#slugs' ) );
```

Comment:
Maybe i'm missing something, but I don't find the product attributes in the settings.
I can find the product categories and tags, but not the attributes 😢

Reply:
You need this check enabled in your test attribute, otherwise there is no archive page and no slug needed.
![image](https://user-images.githubusercontent.com/43221199/167510984-415076c9-2a29-4a31-8d40-331e90dec241.png)

Reply:
Good to now!